src/dataEntryApp/views/subjectDashBoard/components/SubjectDashboardProfileTab.js

```diff
@@ -131,7 +131,11 @@ const SubjectDashboardProfileTab = ({ profile, path, enableReadOnly }) => {
           </ExpansionPanelSummary>
           <ExpansionPanelDetails style={{ paddingTop: "0px" }}>
             {profile.relationships != undefined ? (
-              <GridCommonList gridListDetails={profile.relationships} path={path} />
+              <GridCommonList
+                gridListDetails={profile.relationships}
+                path={path}
```

Comment:
@narendranath-pasupuleti-PSL You are still passing path property here. Did you miss this?